bsip-0044.md

```diff
@@ -172,6 +172,12 @@ Bob has now observed the `preimage` Alice used to "unlock" his HTLC, and he will
 ## **Objects**
 
 ```
+   typedef fc::static_variant<
+      htlc_algo_ripemd160,
+      htlc_algo_sha1,
+      htlc_algo_sha256
```

Comment:
Now you should mention what these three new types are... :-)

Reply:
Do you think it best to put it in a descriptive note below this section or a //comment on each line? I'll give //comment a try.